From 68779e4f272cc285fe76cc437555b5e2db0cb253 Mon Sep 17 00:00:00 2001
From: jinwu <jinwu@codeaurora.org>
Date: Thu, 1 Feb 2018 18:25:31 +0800
Subject: [PATCH 26/68] Fix FC of parsing uri is null

Add judgement to check if uri is null or not.

Change-Id: I95d79253bce67c6a088a79bffd1bbb9a419ec437
---
 src/com/android/gallery3d/util/GDepth.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/gallery3d/util/GDepth.java b/src/com/android/gallery3d/util/GDepth.java
index 877ac049d..39d8db1d2 100644
--- a/src/com/android/gallery3d/util/GDepth.java
+++ b/src/com/android/gallery3d/util/GDepth.java
@@ -64,6 +64,7 @@ public class GDepth {
 
         public boolean parse(Context context, Uri uri) {
             InputStream is = null;
+            if (uri == null) return valid();
             try {
                 is = context.getContentResolver().openInputStream(uri);
                 XMPStream stream = new XMPStream(is);
-- 
2.17.1

